frontend/src/views/Auth/VerifyEmail/VerifyEmail.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import "./VerifyEmail.scss";
+import { Button, Col, Container, Row } from "react-bootstrap";
+import NavBar from "../../../core/components/NavBar";
+
+class VerifyEmail extends Component {
+  // TODO: Integrate with backend
+  state = {
+    email: ""
+  };
+
+  render() {
+    return (
+      <Container fluid id="verify-email-page">
+        <NavBar />
+        <Row>
+          <Col
+            id="main"
+            md={{ span: 8, offset: 2 }}
+            lg={{ span: 4, offset: 4 }}
+          >
+            <h1>
+              We send an email to this address <br />
+              ******23@gmail.com
```

Comment:
This value is returned by the backend when sign up is success.

Reply:
You mean the backend will return the formatted email?

Reply:
I meant to the raw email, but, yes, it is a good idea.